R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -151,16 +142,13 @@ makeTreeSEFromBiom <- function(
         })
         # Create a data.frame from the list
         feature_data <- do.call(rbind, feature_data)
-        # Transposing feature_data and make it DFrame object
-        feature_data <- DataFrame(t(feature_data))
```

Comment:
Is transposing necessary? This led to an error (my data was in correct order already)

Reply:
That's what I was struggling with I don't know how the data looks like when it is loaded as a list [here](https://github.com/microbiome/mia/blob/master/R/makeTreeSummarizedExperimentFromBiom.R#L98C58-L98C58) ? do you have some examples please?